[2024-08-05T00:00:01.590+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-05T00:00:01.617+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_api_data.check_db scheduled__2024-08-04T23:50:00+00:00 [queued]>
[2024-08-05T00:00:01.623+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_api_data.check_db scheduled__2024-08-04T23:50:00+00:00 [queued]>
[2024-08-05T00:00:01.624+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-05T00:00:01.634+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_db> on 2024-08-04 23:50:00+00:00
[2024-08-05T00:00:01.642+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ingest_api_data', 'check_db', 'scheduled__2024-08-04T23:50:00+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/ingest_api_data.py', '--cfg-path', '/tmp/tmpyxft89j5']
[2024-08-05T00:00:01.644+0000] {standard_task_runner.py:91} INFO - Job 124: Subtask check_db
[2024-08-05T00:00:01.645+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=15562) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-05T00:00:01.645+0000] {standard_task_runner.py:64} INFO - Started process 15612 to run task
[2024-08-05T00:00:01.685+0000] {task_command.py:426} INFO - Running <TaskInstance: ingest_api_data.check_db scheduled__2024-08-04T23:50:00+00:00 [running]> on host 52830206e1a2
[2024-08-05T00:00:01.767+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='MS_sIX2Rm@somosyoung.com.br' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingest_api_data' AIRFLOW_CTX_TASK_ID='check_db' AIRFLOW_CTX_EXECUTION_DATE='2024-08-04T23:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-04T23:50:00+00:00'
[2024-08-05T00:00:01.768+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-05T00:00:01.782+0000] {base.py:84} INFO - Using connection ID 'postgres_dw' for task execution.
[2024-08-05T00:00:01.796+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-08-05T00:00:01.800+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-05T00:00:01.801+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-05T00:00:01.810+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ingest_api_data, task_id=check_db, run_id=scheduled__2024-08-04T23:50:00+00:00, execution_date=20240804T235000, start_date=20240805T000001, end_date=20240805T000001
[2024-08-05T00:00:01.858+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-05T00:00:01.881+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-05T00:00:01.883+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
